lib/main.dart

```diff
@@ -99,12 +87,14 @@ class BuildApp extends StatelessWidget {
           ),
           elevatedButtonTheme:
               const ElevatedButtonThemeData(style: ButtonStyle())),
-      home: MyHomePage(),
+      home: const MyHomePage(),
```

Comment:
Change MyHomePage to HomePage

Reply:
HomePage is already used so changed the name to RegistrationClientSplashPage